fe/fe-core/src/main/java/com/starrocks/common/FeNameFormat.java

```diff
@@ -27,15 +27,17 @@
 import com.starrocks.system.SystemInfoService;
 
 public class FeNameFormat {
-    private static final String LABEL_REGEX = "^[-_A-Za-z0-9]{1,128}$";
-    public static final String COMMON_NAME_REGEX = "^[a-zA-Z][a-zA-Z0-9_]{0,63}$";
+    private FeNameFormat() {}
+
+    private static final String LABEL_REGEX = "^[-\\w]{1,128}$";
+    public static final String COMMON_NAME_REGEX = "^[a-zA-Z]\\w{0,63}$|^_[a-zA-Z0-9]\\w{0,62}$";
```

Comment:
You mean the leading underscore format is newly supported? And why multi leading underscores format is not supported, is there any special reason?

Reply:
This feature is required by Celonis.

Reply:
got it , suggested title: [Enhancement] support table name with one leading underscore

Reply:
done